src/libraries/Microsoft.Extensions.Logging.Console/src/SimpleConsoleFormatter.cs

```diff
@@ -15,10 +15,13 @@ internal sealed class SimpleConsoleFormatter : ConsoleFormatter, IDisposable
         private const string LoglevelPadding = ": ";
         private static readonly string _messagePadding = new string(' ', GetLogLevelString(LogLevel.Information).Length + LoglevelPadding.Length);
         private static readonly string _newLineWithMessagePadding = Environment.NewLine + _messagePadding;
-        private static readonly bool _isAndroidOrAppleMobile = RuntimeInformation.IsOSPlatform(OSPlatform.Create("ANDROID"))
-                                                            || RuntimeInformation.IsOSPlatform(OSPlatform.Create("TVOS"))
-                                                            || RuntimeInformation.IsOSPlatform(OSPlatform.Create("IOS"))
-                                                            || RuntimeInformation.IsOSPlatform(OSPlatform.Create("MACCATALYST"));
+#if NETCOREAPP
+        private static readonly bool _isAndroidOrAppleMobile = OperatingSystem.IsAndroid()
```

Comment:
You can make this property now that you are using the more efficient APIs. It works better with trimming.